helpers/if.js

```diff
@@ -77,14 +87,15 @@ function helper(paper) {
             }
         }
 
-        if (options.fn) { // block helper
-            if (result) {
-                return options.fn(this);
-            } else {
-                return options.inverse(this);
-            }
-        } else { // non-block helper
-            return result;
+        if (!options.fn || !options.inverse) {
```

Comment:
This is changing the behavior for `if` when `fn` is not passed. Instead of returning the value, we return true. Is that what we want?

Reply:
No is not, there are tests in place for these cases. This code was added in the wrong way a few moths ago https://github.com/bigcommerce/paper/commit/a12cdbd21c75c4e9391326b4145954d42f941811

I had to change it because it was breaking when `if` is called from `unless` helper because the `unless` helper relies in `fn()` and `inverse()`

Reply:
The return values is always a `boolean`

Reply:
Is it possible to pass `fn` but not `inverse`? Should this be
```
options.fn = typeof options.fn === 'function' ? options.fn : () => true;
options.inverse = typeof options.inverse === 'function' ? options.inverse : () => false;
```

Reply:
both need to exist. If one of them is missing, is probably a bug